.github/workflows/test.yml

```diff
@@ -109,6 +118,7 @@ jobs:
           - jre
           - maven
           - tomcat
+          - busybox
```

Comment:
Indentation seems off

Reply:
I'm not seeing it, and it looks like GHA isn't? https://github.com/chainguard-dev/terraform-provider-imagetest/actions/runs/13641479524/job/38132197388?pr=309